src/loading/loading.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, Input } from "@angular/core";
+
+@Component({
+	selector: "ibm-loading",
+	template: `
+		<div class="bx--loading-overlay">
+			<div data-loading class="bx--loading">
+				<svg class="bx--loading__svg" viewBox="-75 -75 150 150">
+					<title>{{title}}</title>
+					<circle cx="0" cy="0" r="37.5" />
+				</svg>
+			</div>
+		</div>`,
+	styles: []
+})
+export class LoadingComponent {
+	@Input()
+	title = "Loading";
+}
+
+@Component({
+	selector: "ibm-small-loading",
+	template: `
+		<div data-loading class="bx--loading bx--loading--small">
```

Comment:
FWIW `data-loading` is for vanilla component instantiation

Reply:
If styling works the same without it, drop it :)

Container `div`s should normally be dropped and attributes assigned to the component element with `HostBinding`. It creates fewer elements in the DOM and makes apps with better performance.